docs/reference/search/rrf.asciidoc

```diff
@@ -1,6 +1,423 @@
 [[rrf]]
 === Reciprocal rank fusion
 
-Reciprocal Rank Fusion (RRF) is a simple method to combine document result sets
-from multiple queries where the queries' document scores may be unrelated.
+NOTE: Reciprocal rank fusion requires a Platinum subscription or higher.
 
+IMPORTANT: Reciprocal rank fusion is in technical preview. Features in technical
+preview may be changed or removed in a future release. Elastic will apply best
+effort to fix any issues, but features in technical preview are not subject to
+the support SLA of official GA features.
+
+Reciprocal rank fusion (RRF) is a method for combining results from
+multiple queries with different relevance indicators into a single
+result set where there is no requirement the relevance indicators for
+each query are related to each other in order to achieve high-quality results.
```

Comment:
nit: may break this paragraph into several sentences?

Reply:
Should we reference the [original paper](https://plg.uwaterloo.ca/~gvcormac/cormacksigir09-rrf.pdf) that came up with this method?

Reply:
Also would be nice to add a little bit more what is a rational behind this combination, may be something like this:  "Combining search results from several retrieval methods can yield better performance than from a single one. But scores of these results are often be of different scale (e.g. knn search and BM25), so it could be challenging to combine them linearly.  RRF disregards scores and  only considers rank of a document in a result set, making combination of results with scores of different scales easier. "

Reply:
I will work on improving this. I'm trying to figure out if we have a policy for adding white papers.

Reply:
I added the link to the paper and tried to clean this up a bit as per your suggestions.